test/e2e/storage/pv_protection.go

```diff
@@ -85,6 +85,8 @@ var _ = utils.SIGDescribe("PV Protection", func() {
 		By("Checking that PV Protection finalizer is set")
 		pv, err = client.CoreV1().PersistentVolumes().Get(pv.Name, metav1.GetOptions{})
 		Expect(err).NotTo(HaveOccurred(), "While getting PV status")
+
+		// NOTE: If the following Expect is failed, please check StorageObjectInUseProtection plugin of admission controller on k8s cluster.
```

Comment:
Maybe we should improve the kubernetes [documentation on admission controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) and also consider enabling the controller by default.

I'm not sure if the e2e test is the right place to document this.

Reply:
hehe, I just wanted to avoid creating the similar PR which adds timeout loop by reading the comment in the future.

Reply:
Or can we enable the plugin by default?